question_eleven.py

```diff
@@ -0,0 +1,50 @@
+import math
+def common_diff(seq):
+    return seq[1] - seq[0]
+
+def correct_ap(seq, common_diff):
+    correct_se = [seq[0]]
+    for i in range(1, len(seq)):
+        correct_se.append(correct_se[-1] + common_diff)
+    return correct_se
+
+ap_seq = [2, 5, 8, 11, 15, 17]
+
+common_diff = common_diff(ap_seq)
+
+ap = correct_ap(ap_seq, common_diff)
+
+print("A.P. =", ap)
+
+
+def common_gp_diff(seq):
+    return seq[1] / seq[0]
```

Comment:
This is not a difference. This is a ratio. Please rename function to `common_ratio`.